odl/solvers/nonsmooth/proximal_operators.py

```diff
@@ -138,17 +142,26 @@ def convex_conj_prox_factory(sigma):
 
         Parameters
         ----------
-        sigma : positive float
-            Step size parameter
+        sigma : positive float or componentwise positive space element
```

Comment:
pointwise

This should fit on one line. How about just speaking of `positive float or array-like` and describe what cases of arrays are allowed?